### PR TITLE
added comments to "it"

### LINKED ISSUE #29: Add comment to it as argument
So like in rspec we could use comment for it:
```ruby
it 'should fail' do
    expect(1).to eq 7
end

it('should fail') { expect(1).to eq 7 } 

it('should pass', expect(1).to eq 1)
```

### PR DESCRIPTION
**Pull request by fartem**
I think I understood the idea correctly, but open to other views.

Closed #29.

### CODE REVIEW

File: test/comment_spec.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+BDDSM.describe 'Test case with comments' do
```
Comment (sergio-fry) [inline test/comment_spec.rb:1]: This test will  fail so, build will be red after #34 is merged.

Comment (fartem) [reply test/comment_spec.rb:1]: I will fetch it after merge and retest this again.

Comment (sergio-fry) [reply test/comment_spec.rb:1]: merged

File: lib/bddsm/describe.rb [hunk 2]
```diff
@@ -12,7 +12,8 @@ def run
       instance_eval(&@block)
     end
 
-    def it(&)
+    def it(comment = nil, &)
+      @comment = comment
```
Comment (sergio-fry) [inline lib/bddsm/describe.rb]: Are we going to have single comment to all 'its' in a describe block? Looks like we should have separated

Comment (fartem) [reply lib/bddsm/describe.rb]: Do you mean you need to comment out the remaining blocks?

Comment (sergio-fry) [reply lib/bddsm/describe.rb]: No. If we have multiple 'it'-s in a describe block, the last 'it' should override `@comment`, so the only one comment is going to be stored. 

```ruby
describe "test" do
  it "this one is going to be removed" do
  end

  it "this one will be saved" do
  end
end
```

Comment (fartem) [reply lib/bddsm/describe.rb]: Ah, I understand. I'll fix it soon.

**Review by sergio-fry — CHANGES_REQUESTED** [review]
build is failing

**Comment by fartem** [comment]
> build is failing

Build failed because after last commit in `main` we have exit code with error.

**Comment by fartem** [comment]
@sergio-fry right now on every push we have gem publish action. Maybe I will fix it in another PR?

**Review by sergio-fry — APPROVED** [review]

**Pull request merged**